### DISCUSSION TITLE
This function GeneIntegrationMatrix requires scATAC and  scRNA from same sample?

### DISCUSSION THREAD

**Discussion by xiezhan** [GreenleafLab/ArchR#865] [Questions / Documentation]
This function, will integrate multiple subsets of scATAC cells
with a scRNA experiment, compute **_matched_** scRNA profiles and then
store this in each samples ArrowFile.

How to understand the "matched" scRNA profiles ?

thanks

**Comment by rcorces** [comment]
Please note that the Issues section is not meant for usage questions. Those belong in Discussions as clearly indicated when you try to open an issue:
![image](https://user-images.githubusercontent.com/10343046/124215763-8a53ad80-daa9-11eb-8a4b-3214834ee8a6.png)
I have migrated your post there.

matched just means from the same type of sample. it could be the exact same sample or it could be publicly available data from the same cell types / tissue type.

**Reply by xiezhan** [reply]
Thanks so much！

Another question I met when i use ArchR，i used BAM file from  cellranger-atac to creat arrow file，i got an error，exactly the same error with https://github.com/GreenleafLab/ArchR/issues/292，

I found this from  ArchR manual：

**“The input format used will depend on the pre-processing pipeline used. For example, the 10x Genomics Cell Ranger software returns fragment files while sci-ATAC-seq applications would use BAM files. ”**

What‘s the meanings of the sentences？ 

Only sci-ATAC data can use BAM as input？10x Genomics can't？10x Genomics only use Fragment？

thanks

**Reply by rcorces** [reply]
You might want to brush up on your scATAC-seq applications and pipelines if that sentence is confusing.
If you've run cell-ranger, you should use the fragments file as input to ArchR. it is much more straightforward than using bam files.

**Reply by xiezhan** [reply]
you mean  BAM  files  can also be  used  when I run cellranger, though BAM files might be not straightforward?

**Reply by rcorces** [reply]
correct

**Reply by xiezhan** [reply]
ok  thanks  so much
